lib/ual.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+class SunbeamUAL {
+  constructor (opts) {
+    this.opts = Object.assign({
+      appName: 'Eosfinex-UAL',
+    }, opts)
+
+    this.account = null
+    this.user = this.opts.user
+  }
+
+  async auth (client, opts) {
+    if (!this.account) {
+      const accountName = await this.user.getAccountName()
+
+      const { network } = opts

```

Comment:
`network` variable is not used anywhere, should it be here?

Reply:
removed